### PR TITLE
add jupytext to env and update lock files

### PR DESCRIPTION
**Pull request by JessicaS11**
Hopefully will resolve failing checks in #12.

### CODE REVIEW

**Comment by jomey** [comment]
Looks like it is still stumbling on the `jinja-markdown` requirement.
Did this build locally from scratch for you?

**Comment by JessicaS11** [comment]
I didn't try building locally - only updating the env/lock files.

File: conda/conda-osx-64.lock.yml [hunk 1]
```diff
@@ -374,8 +376,8 @@ dependencies:
   - jupyterlab=3.4.2=pyhd8ed1ab_0
   - ipyleaflet=0.16.0=pyhd8ed1ab_4
   - jupyter-sphinx=0.3.2=pyhd8ed1ab_1
-  - jupyterhub-singleuser=2.3.0=pyhd8ed1ab_0
+  - jupyterhub-singleuser=2.3.1=pyhd8ed1ab_0
   - myst-nb=0.13.2=pyhd8ed1ab_0
   - jupyter-book=0.12.3=pyhd8ed1ab_0
   - pip:
-    - jinja-markdown === 1.210911 
+    - jinja-markdown === 1.210911 --hash=md5:None
```
Comment (jomey) [inline conda/conda-osx-64.lock.yml]: This `--hash` change currently breaks the build.
I am looking into this on why this was added by the script

Comment (jomey) [reply conda/conda-osx-64.lock.yml]: Looks like [this step](https://github.com/snowex-hackweek/website2022/blob/d49ed4d8d8fc3f6bb82a8fba6408318ac9e2c58c/conda/lock-environment.sh#L35) in the `lock-environment.sh` file did succeed for you locally.

I will manually remove this hash which should fix this build

**Review by jomey — APPROVED** [review]

File: conda/conda-osx-64.lock.yml (hunk 1 above)
Comment (JessicaS11) [reply conda/conda-osx-64.lock.yml]: I actually got an illegal flag note from sed on this line (sorry I didn't catch that in the output before):
```
sed: illegal option -- r
usage: sed script [-Ealn] [-i extension] [file ...]
       sed [-Ealn] [-i extension] [-e script] ... [-f script_file] ... [file ...]
rm: *.yml.bak: No such file or directory
```

Comment (jomey) [reply conda/conda-osx-64.lock.yml]: I assume you are on a Mac?

Comment (JessicaS11) [reply conda/conda-osx-64.lock.yml]: Yup - that explains it!

**Pull request merged**